core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -69,45 +63,60 @@
                 (despersonalize)="onDespersonalize($event)"
                 data-testId="persona-selector" />
             <dot-edit-ema-workflow-actions
+                *ngIf="es.editorData.mode !== editorMode.VARIANT"
                 [inode]="es.editor.page.inode"
                 (newPage)="handleNewPage($event)" />
         </ng-container>
+        <ng-container bottom>
+            <dot-ema-info-display
+                *ngIf="
+                    es.editorData.mode === editorMode.DEVICE ||
+                    es.editorData.mode === editorMode.VARIANT
+                "
```

Comment:
A lot of `||` in this template... should we rethink the state structure?

Reply:
At the time I don't have more ideas than to do a pipe an pass an array of constrains

Reply:
something like 

`["SOME_MODE", "ANOTHER_MODE"].find((mode) => mode === currentMode)`

but encapsulated in a pipe.